src/sdk/main/src/AccountCreateTransaction.cc

```diff
@@ -51,7 +51,7 @@ AccountCreateTransaction::AccountCreateTransaction(
   const std::map<TransactionId, std::map<AccountId, proto::Transaction>>& transactions)
   : Transaction<AccountCreateTransaction>(transactions)
 {
-  setDefaultMaxTransactionFee(Hbar(5LL));
+  setDefaultMaxTransactionFee(Hbar(10000LL));
```

Comment:
The changes to the default max transaction fees for `AccountCreateTransaction` are to make the TCK tests for max auto token associations work. I have posed a question about how we want to proceed with this, whether we want to up the default max transactions here or they should be set manually.

Reply:
Can we resolve and merge this or are there still discussions around this?

Reply:
I removed this and put a temporary solution on the TCK which will set the max transaction fee manually. That will be removed when support for [HIP-904](https://hips.hedera.com/hip/hip-904) is added, which will make it so the account creator doesn't pay for automatic token association slots anymore.